index.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+$DS = DIRECTORY_SEPARATOR;
+
+require 'constants.php';
+require "libraries{$DS}console_helpers.php";
+require "libraries{$DS}helpers.php";
+require "app{$DS}HistoryRepository.php";
```

Comment:
this files are not using in the index.php, please incude files only in class when it will be used